pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -935,15 +937,24 @@ func appendStatusCauseForCNIPluginHasNoNetworkName(field *k8sfield.Path, incomin
 	return causes
 }
 
-func validateNetworkHasOnlyOneType(field *k8sfield.Path, cniTypesCount int, causes []metav1.StatusCause, idx int) []metav1.StatusCause {
+func validateNetworkHasOnlyOneType(field *k8sfield.Path, network *v1.Network, causes []metav1.StatusCause, idx int) []metav1.StatusCause {
+	cniTypesCount := 0
+	if network.Pod != nil {
+		cniTypesCount++
+	}
+	if network.Multus != nil {
+		cniTypesCount++
+	}
+
```

Comment:
I wonder if something like this will be more readable:
```suggestion
	
	if network.Pod != nil && network.Multus != nil {
		return append(causes, metav1.StatusCause{
			Type:    metav1.CauseTypeFieldValueRequired,
			Message: "should have only one network type",
			Field:   field.Child("networks").Index(idx).String(),
		})
	}
	else if network.Pod == nil && network.Multus == nil {
	    return append(causes, metav1.StatusCause{
			Type:    metav1.CauseTypeFieldValueRequired,
			Message: "should have a network type",
			Field:   field.Child("networks").Index(idx).String(),
		})
	}
```

Reply:
Done